src/encoded/static/components/static-pages/placeholders/index.js

```diff
@@ -6,16 +6,19 @@ import memoize from 'memoize-one';
 import _ from 'underscore';
 
 import { SlideCarousel } from './SlideCarousel';
+import { PageCarousel } from './PageCarousel';
 import { JointAnalysisMatrix } from './JointAnalysisMatrix';
 
-export { SlideCarousel, JointAnalysisMatrix };
+import { console } from '@hms-dbmi-bgm/shared-portal-components/es/components/util';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['console' is defined but never used. (no-unused-vars)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4476879)